### PR TITLE
chore(snapshots): update all prefixes to `c4d`

### PR DESCRIPTION
**Pull request by kennylam**
### Description

Update all Dotcom prefixes to `c4d`.

### Changelog

**New**

- {{new thing}}

**Changed**

- {{changed thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: packages/services/src/services/Translation/Translation.js [hunk 1]
```diff
@@ -35,8 +35,8 @@ const _ddsEndpointDefault =
  */
 const _ddsEndpoint =
   (process &&
-    (process.env.REACT_APP_DDS_TRANSLATION_ENDPOINT ||
-      process.env.DDS_TRANSLATION_ENDPOINT)) ||
+    (process.env.REACT_APP_C4D_TRANSLATION_ENDPOINT ||
+      process.env.C4D_TRANSLATION_ENDPOINT)) ||
   _ddsEndpointDefault;
```
Comment (annawen1) [inline packages/services/src/services/Translation/Translation.js:40]: probably have to change the prefix for the `_ddsEndpoint` and `_ddsEndpointDefault` instances too?

File: packages/web-components/IMPLEMENTATION_NOTES.md [hunk 2]
```diff
@@ -97,7 +97,7 @@ Most components use the following to define CSS:
 import styles from './some-component.scss';
 
 @customElement('dds-some-component')
```
Comment (annawen1) [inline packages/web-components/IMPLEMENTATION_NOTES.md:99]:
```suggestion
@customElement('c4d-some-component')
```

File: packages/web-components/IMPLEMENTATION_NOTES.md [hunk 3]
```diff
@@ -446,11 +446,11 @@ And container components are created by `ConnectMixin`:
 
 ```typescript
 @customElement(`${ddsPrefix}-footer-container`)
```
Comment (annawen1) [inline packages/web-components/IMPLEMENTATION_NOTES.md:448]:
```suggestion
@customElement(`${c4dPrefix}-footer-container`)
```

File: packages/web-components/src/components/pricing-table/pricing-table-group.ts [hunk 4]
```diff
@@ -9,18 +9,18 @@
 
 import settings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
 import StableSelectorMixin from '../../globals/mixins/stable-selector';
-import DDSStructuredListGroup from '../structured-list/structured-list-group';
+import C4DStructuredListGroup from '../structured-list/structured-list-group';
 import styles from './pricing-table.scss';
 import { carbonElement as customElement } from '../../internal/vendor/@carbon/web-components/globals/decorators/carbon-element';
 const { stablePrefix: ddsPrefix } = settings;
```
Comment (annawen1) [inline packages/web-components/src/components/pricing-table/pricing-table-group.ts:15]: probably gotta do a massive find and replace of `ddsPrefix` as well

```suggestion
const { stablePrefix: c4dPrefix } = settings;
```

File: packages/services/src/services/Translation/Translation.js (hunk 1 above)
Comment (kennylam) [reply packages/services/src/services/Translation/Translation.js:40]: Good catch!

**Comment by IgnacioBecerra** [comment]
LGTM! Seems like this file is the only one missing:

Screenshot 2023-08-31 at 11 05 06 PM

**Review by IgnacioBecerra — APPROVED** [review]
Just a couple conflicts needing fixing but otherwise LGTM!

**Pull request merged**